### PR TITLE
Create pages in segmented campaigns

### PR DESCRIPTION
**Pull request by rowanhogan**
Create MDS values for custom questions after page is created
Update login/signup routes to handle redirects instead of child components

### CODE REVIEW

File: source/store/sessionPage/index.js [hunk 1]
```diff
@@ -43,12 +66,32 @@ export const createUserPage = (data) => (dispatch) => {
...
+    campaignDate: data.eventDate
   }).then((json) => {
-    dispatch(createUserPageSuccess(json))
-    return Promise.resolve(json)
+    const createKey = (key, value) => {
+      return axios.post(`${process.env.METADATASERVICE_URL}/api/v1/metadata`, {
+        token: process.env.MD5_TOKEN,
+        type: 'Page',
+        id: json.page.id,
+        key: key,
```
Comment (kizza) [inline source/store/sessionPage/index.js]: You can just say `key` here (and `value` below) if you wish... you may prefer to be explicit :)

File: source/store/sessionPage/index.js [hunk 2]
```diff
@@ -43,12 +66,32 @@ export const createUserPage = (data) => (dispatch) => {
...
+        value: value
+      })
+    }
+
+    return Promise.all([
```
Comment (kizza) [inline source/store/sessionPage/index.js]: :+1:

File: .env.production [hunk 3]
```diff
@@ -0,0 +1,13 @@
+PRISMIC_REPO=beyondblue-bash
+SUPPORTER_URL=https://everydayhero.com
+APPLICATION_URL=https://beyondbluefundraising.org.au
+API_CLIENT_ID=
+METADATASERVICE_URL=https://mds.everydayhero.io/
+CHARITY_UID=au-899
+MD5_TOKEN=9c0f5d3d9eb8ea7df688a0495898d62d
```
Comment (kizza) [inline .env.production]: Should this be a genuine secret?  Not sure how secrets are provisioned for these "landing pages / microsites"... it probably doesn't matter...

Comment (kizza) [reply .env.production]: ...talked to Ellis IRL... apparently there is no provisioning for genuine secrets.  It does mean anyone could post any keys under our token... but that said, it's in the client already!  So there is no option. Ignore :)

Comment (rowanhogan) [reply .env.production]: I actually added this to reduce a runtime dependency... Whilst not explicitly publicised previously, we were always using the same _secret_ value, which is an MD5 token based on the charity UID. So as it wasn't particularly secure as it was, I thought it was worth simplifying its' includsion.

In regards to how we usually manage secrets, we'd keep them out of repos of course and use Buildkite to manage non-dev secrets

Comment (kizza) [reply .env.production]: :+1:

**Pull request merged**